rest/route/github.go

```diff
@@ -389,9 +389,9 @@ func (gh *githubHookApi) handleGitTag(ctx context.Context, event *github.PushEve
 	foundVersion := map[string]bool{}
 
 	const (
-		checkVersionAttempts      = 5
-		checkVersionRetryMinDelay = 100 * time.Millisecond
-		checkVersionRetryMaxDelay = 3 * time.Second
+		checkVersionAttempts      = 10
```

Comment:
I'd like to not double the number of attempts for now; for repos with a large number of branch projects, this already results in a high number of queries. Is there a way to get the same 30 second result with just 5 attempts?